cmake/Modules/FindVPL.cmake

```diff
@@ -44,7 +44,11 @@ The following cache variables may also be set:
 
 find_package(PkgConfig QUIET)
 if(PKG_CONFIG_FOUND)
-  pkg_search_module(_VPL QUIET vpl)
+  pkg_search_module(_VPL IMPORTED_TARGET QUIET vpl)
+  if(${_VPL_FOUND})
```

Comment:
You can use `if(_VPL_FOUND)` here.

Reply:
Also if I understood the issue right, this is just an issue in specific distributions and is fixed by later variants. As this circumvents the entire finder (and also the feature summary) I'd rather have this workaround limited to those "bad" distributions then.

Reply:
Ace mentioned we just use one recipe for all of ubuntu, so its difficult to implement that there. In this case its just more correct then the rest of our finder so I think its fine to do this everywhere, even where pkg-configs are not formatted weirdly.